src/main/java/picard/sam/SamErrorMetric/ReadBaseStratification.java

```diff
@@ -481,6 +487,39 @@ public String getSuffix() {
         }
     }
 
+
+    /**
+     * Stratifies base based on location within each tile
+     */
+    public static class FlowCellLocationStratifier extends RecordStratifier<String> {
+        final int locationBinSize;
+        private static OpticalDuplicateFinder opticalDuplicateFinder = new OpticalDuplicateFinder();
+
+        @Override
+        public String stratify(final SAMRecord sam) {
+            try {
+                final PhysicalLocation location = new PhysicalLocationInt();
+                opticalDuplicateFinder.addLocationInformation(sam.getReadName(), location);
+                String tile = Integer.toString(location.getTile());
+                String x = Integer.toString(location.getX() / LOCATION_BIN_SIZE);
+                String y = Integer.toString(location.getY() / LOCATION_BIN_SIZE);
+                return (tile + "_" + x + "_" + y);
```

Comment:
since there already is a tile stratifier, it woudl seem more consistent to just add x & y stratifier (or one stratifiyer for x and one for y?) and then use it in conjunction with the Tile stratifier

Is there a reason this cannot work?

Reply:
Not that I know of, I can try it out and see how it compares (in terms of speed) to the way it is configured currently